### PR TITLE
CNV TargetCoverageSexGenotyper: tag doc, give example command and add notes from MB's email

### PR DESCRIPTION
**Pull request by sooheelee**
Much smoothing to be desired but this is what it is right now.

### CODE REVIEW

**Comment by sooheelee** [comment]
The check failed because it timed out.

File: src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java [hunk 1]
```diff
@@ -22,7 +23,7 @@
 import java.util.List;
 
 /**
- * A command line tool for inferring sex genotypes from a tab-separated raw target coverage file.
+ * Infers sex genotypes from raw target coverage and contig annotations.
```
Comment (mbabadi) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java]: ... "from raw target coverage data and germline contig ploidy annotations."

File: src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java [hunk 2]
```diff
@@ -38,29 +39,51 @@
...
+ *     This recommendation applies especially to whole genome sequencing (WGS) samples.
+ * </p>
+ * <h3>Example</h3>
+ *
+ * <pre>
+ * java -Xmx4g -jar $gatk_jar TargetCoverageSexGenotyper \
+ *   --input combined_read_counts.tsv \
+ *   --contigAnnotations grch37_contig_annotations.tsv \
+ *   --targets targets.tsv \
+ *   --output SEX_GENOTYPES.tsv
```
Comment (mbabadi) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java]: --output inferred_sex_genotypes.tsv

File: src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java [hunk 3]
```diff
@@ -103,23 +126,24 @@
     protected File outputSampleGenotypesFile;
 
     @Argument(
-            doc = "Input contig annotations file.",
+            doc = "Input contig annotations file. For an example file, see the GATK Resource Bundle.",
```
Comment (mbabadi) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java]: "Input germline contig ploidy annotations file. ..."

File: src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java (hunk 1 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java]: incorporated

File: src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java (hunk 2 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java]: changed

File: src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java (hunk 3 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java]: Incorporated

**Comment by sooheelee** [comment]
@mbabadi I've incorporated your feedback.

**Comment by sooheelee** [comment]
There will be a chance for discussion and review in the second iteration of edits. For now, need to merge as many PRs as possible since gatk4 and gatk-protected repos are being merged today.

**Pull request merged**